docs/rules/no-var-requires.md

```diff
@@ -0,0 +1,28 @@
+# Disallows the use of require statements except in import statements (no-var-requires)
+
+In other words, the use of forms such as `var foo = require("foo")` are banned. Instead use ES6 style imports or `import foo = require("foo")` imports.
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```ts
+require("foo");
```

Comment:
This should be allowed IMO since there isn’t an equivalent with the `import =` syntax.

Reply:
Yeah, I could see how introducing this as-is would be a breaking change, since all standalone `require()` calls would be flagged by this rule.

I will push up a fix that allows this. If we ultimately want to disallow it before merging, I will just revert the commit. 👍